unreleased/issue-13690-adjust-exported-funcs.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/telemetryquerylanguage
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Unexport `BoolExpressionEvaluator`, `Literal`, `NewGetter`, `PathExpressionParser`, `EnumParser`, and `NewFunctionCall`.
```

Comment:
You'll want to remove `PathExpressionParser` and `EnumParser` from this list.

Reply:
You're right, thanks